index.js

```diff
@@ -106,7 +106,7 @@ exports.sendData = (ipcRenderer, processName, values) => {
   });
 };
 
-exports.onRecieveData = (ipcRenderer, processNameOuter, func) => {
+exports.onReceiveData = (ipcRenderer, processNameOuter, func) => {
```

Comment:
Thanks for bringing that up. I will have to rectify this at a few other places as well.

Reply:
Will merge this soon.

Reply:
Since this means a breaking change, the major version could just be increased. I have forked the repo and done so while making some changes that were necessary to upgrade Electron for https://github.com/fossasia/pslab. I am offering a second PR. You can safely take @2secslater's PR first and my patch would just be skipped.

Reply:
@AakashMallik Just merged your PR. That was a great help. I would try to update the docs as well but I am a little caught up with stuff. Let me know if you would like to update the docs from ur end.